tests/draft7/ref.json

```diff
@@ -818,5 +818,71 @@
                 "valid": false
             }
         ]
+    },
+    {
+        "description": "ref to if",
+        "schema": {
+            "$ref": "http://example.com/ref/if",
```

Comment:
The `$ref`s for these need to be wrapped in `allOf` because it's draft-07.

Reply:
Indeed, fixed, https://github.com/json-schema-org/JSON-Schema-Test-Suite/commit/e4e1a220bfde13ae37cc0d8c1ff14193e2a33d4d thanks!